### DISCUSSION TITLE
google calendar events won't show in spiral linux gnome

### DISCUSSION THREAD

**Discussion by quadransmuralis** [SpiralLinux/SpiralLinux-project#348] [Q&A]
Hello, I've just installed SpiralLinux with the Gnome desktop.
I can't get my google calendar events to appear in the top panel calendar.
(I can see my google account in files.)
If anyone could help I'd be grateful.
Thx.

**Comment by geckolinux** [comment]
Hi there, in what other distributions has that feature worked for you? I think it might need one of the `evolution-*` packages for that functionality.

**Reply by quadransmuralis** [reply]
Well, it works with debian 12 with the gnome desktop. It also works with linux mint.

**Reply by geckolinux** [reply]
OK good. Please check which `evolution-*` packages are installed on those systems and let me know.
